InMobi/InMobiAdapter/ALInMobiMediationAdapter.m

```diff
@@ -7,9 +7,9 @@
 //
 
 #import "ALInMobiMediationAdapter.h"
-@import InMobiSDK;
+#import <InMobiSDK/InMobiSDK.h>
```

Comment:
why there is change in header as MAX adapter is already in Swift

Reply:
they have again done this change, that's why provided this reference link .
https://github.com/InMobi/AppLovin-MAX-SDK-iOS/pull/4/files